### ISSUE TITLE
The difference between your pretrained model and that generated from your script?

### ISSUE THREAD

**Issue by yq1011** [anewell/pose-hg-train#24]
Hi!

Thanks for your shared code. I find that there is a little bit differences between your pretrained model and the model I generate from your code.

There is a 1GB cuda memory gap during training between them. 

Could you tell me what could be the reason of that?

Thanks!

**Comment by yq1011** [comment]
BTW, what's the valid set for your pretrained model? I set the validImgs as 1000 and the valid accuracy is so high.

I guess I should set the validImgs as 3000, just as your previous opt setting?

**Comment by anewell** [comment]
The pretrained model is the version used for the paper, but I have since added [an extra residual module](https://github.com/anewell/pose-hg-train/blob/master/src/models/hg.lua#L53) which introduces extra computation at a high resolution. That module is responsible for the memory difference you are seeing, but it should help performance.

As for the validation set, that was my bad for changing things up and having the code initialize a random validation split. I have just pushed an update that will instead choose the split used in our paper by default. But if you do want to initialize a random validation split, you can now use the flag '-randomValid'.

**Comment by yq1011** [comment]
I see, thanks!
